jinja2cli/cli.py

```diff
@@ -178,6 +178,8 @@ def cli(opts, args):
 
     output = render(template_path, data, extensions)
 
+    if isinstance(output, bytes):
```

Comment:
I guess we can. I think it would be easier to do by using six. Would it be OK to add it as a dependency? If not, no big deal, you solution is simple enough.